rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -167,11 +167,17 @@ BoolFalse.  BoolLiteral ::= "false" ;
 -- Ground types:
 -- The "Literal" suffix avoids collisions with Simple Types
 GroundBool.    Ground ::= BoolLiteral ;
+GroundBigInt.  Ground ::= "BigInt" "(" BigIntLiteral ")" ;
 GroundInt.     Ground ::= LongLiteral ;
 GroundString.  Ground ::= StringLiteral ;
 GroundUri.     Ground ::= UriLiteral ;
 token StringLiteral ( '"' ((char - ["\"\\"]) | ('\\' ["\"\\nt"]))* '"' );
 token UriLiteral ('`' ((char - ["\\`"]) | ('\\' ["`\\"]))* '`') ;
+
+-- BigInt
+BigIntPositive. BigIntLiteral ::= LongLiteral ;
+BigIntNegative. BigIntLiteral ::= "-" LongLiteral ;
```

Comment:
We don't actually have `nagative Int` before. `-` is just a normal operator. Maybe we should add `Negative Int`, too.